backend/src/test/java/com/example/backend/controller/CarHealthControllerTest.java

```diff
@@ -39,30 +32,39 @@ class CarHealthControllerTest {
     @Autowired
     private MockMvc mockMvc;
 
-    @MockBean
+    @Autowired
+    private CarHealthRepository carHealthRepository;
+    @Autowired
     private CarHealthService carHealthService;
 
-    @MockBean
+    @Autowired
+    private WorkRepository workRepository;
+    @Autowired
     private WorkService workService;
 
-    @MockBean
+    @Autowired
+    private WorkTypeRepository workTypeRepository;
+    @Autowired
+    private WorkTypeService workTypeService;
+
+    @Autowired
     private IdGeneratorService idGeneratorService;
 
     @Autowired
     private ObjectMapper objectMapper;
 
+    @DirtiesContext
```

Comment:
You can also put @DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
on top of the whole test. Then you don't have to repeat it for every single test.